.github/workflows/build.yml

```diff
@@ -84,7 +94,8 @@ jobs:
             git checkout $commit
             rm -rf build/
             set +e
-            west -v build -b ${{ matrix.board }} ${{ matrix.path }} -p
+
```

Comment:
Let's not add a new line here unless there is a good reason for it. Let's minimize the delta. If you are changing this for pure cosmetic reason, this should be in a separate commit.